http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -1104,19 +1238,19 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
     }
   }
 
-  private[util] def maybeCreateWsmIdentityAndDatabase(app: App,
-                                                      workspaceId: WorkspaceId,
-                                                      landingZoneResources: LandingZoneResources,
-                                                      namespace: KubernetesNamespace
+  private[util] def maybeCreateWsmIdentityAndDatabases(app: App,
```

Comment:
This function was previously only used to create an identity and database for WDS (in LZ postgres), but since we want to create Workflows-App in the same manner it has been generalized to create an identity and database(s) depending on the app type, and should be flexible to new apps with shared databases.

Reply:
Is this method specific to shared databases? Maybe we should name it as such: `maybeCreateSharedDatabases` or something.

Reply:
Another note: this implementation might change (not sure exactly how) once Leo adopts https://github.com/DataBiosphere/terra-workspace-manager/pull/1430, which changes some of the WSM APIs used for creating identities, databases, and namespaces.

Reply:
Is `maybeCreateWsmIdentityAndSharedDatabases` too much of a mouthful? I think it's important to note the identity is created within this method as well

Reply:
That sounds fine… mainly I was just confused what this method was doing until I realized it handle all our shared database creation. Whereas `maybeCreateCromwell` databases creates all the user-private DBs needed by cromwell apps.